code/user_settings.py

```diff
@@ -0,0 +1,148 @@
+from talon import Module, fs, Context
+import os
+import csv
+from pathlib import Path
+from typing import Dict, List, Tuple
+import threading
+
+
+# NOTE: This method requires this module to be one folder below the top-level
+#   knausj folder.
+SETTINGS_DIR = Path(__file__).parents[1] / "settings"
```

Comment:
why is this parents[1] instead of parents[2]?

parents[1] puts it inside the knausj_talon git repository, which seems undesirable.

On the other hand, parents[2] puts it as a sibling. I guess for people who are using knausj_talon as their user/ directory, parents[2] would put it directly in the .talon/ directory instead of in .talon/user/. is this likely to be a problem?

is there some way of getting our hands on the .talon/user/ directory directly? that seems like the right thing to do here.

Reply:
EDIT: `actions.path.talon_user()` produces the user directory

Reply:
(Note for anyone reading, this was addressed on Slack)